app/[locale]/run-a-node/_components/run-a-node.tsx

```diff
@@ -474,7 +474,7 @@ const RunANodePage = ({
               </ul>
             </div>
             <ButtonContainer>
-              <ButtonLink href="https://shop.dappnode.io/">
+              <ButtonLink href="https://dappnode.io/">
```

Comment:
thanks @Co1nB3e for surfacing! Looks like the shop is here:

```suggestion
              <ButtonLink href="https://dappnode.com/collections/frontpage">
```